### DISCUSSION TITLE
GNS3 Error on MacBook M2 Pro

### DISCUSSION THREAD

**Discussion by ashanendra** [community/community#112220] [Discussions]
### Select Topic Area

Question

### Body

GNS3 Version: GNS3 v2.2.46
System: MacBook M2 Pro Software Sonoma 14.4 (23E214) 
Operating System: MacOS
VMware Fusion: Professional Version 13.5.1

I have tried to run Cisco FTD or Cisco ASAv Appliance Image on my GNS3 then getting error-

"Error" - The selected server does not support KVM. A Linux server or the GNS3 VM running in VMware is required.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ashanendra ! We're happy you're here. The Discussions category is solely related to conversations around the GitHub product Discussions. 

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Apple products like their [Mac Community](https://discussions.apple.com/community/mac). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
